pkg/plugins/policies/matchers/match_test.go

```diff
@@ -98,6 +98,9 @@ var _ = Describe("Match", func() {
 			}
 
 			for num, tc := range testCaseMap {
+				if num != "01" {
+					break
+				}
```

Comment:
Why this is there?

Reply:
I forgot to clean this up 🤦‍♂️ that's how I do `FEntry` when entries are auto-generated